index.html

```diff
@@ -0,0 +1,307 @@
+          <li><a href="#">News</a></li>
+          <li>
+            <a
+              href="#"
+              class="text-callAction font-semibold border-4 p-2 border-callAction"
+              >Buy Ticket</a
+            >
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Hero -->
+    <section
+      id="hero"
+      class="bg-bgHero md:bg-bgHeroDesktop bg-no-repeat bg-cover min-h-[600px]"
+    >
+      <div class="container mx-auto px-[2%] py-24">
+        <div class="flex flex-col space-y-12">
+          <div class="flex flex-col gap-2">
+            <span class="text-callAction text-xl font-medium md:text-3xl"
+              >"Hello! Brillian Minds"
+            </span>
+            <h1
+              class="text-4xl md:text-6xl md:w-[90%] lg:w-[70%] text-callAction font-[900] bg-textImg image-text my-font"
+            >
+              The EX-INVESTOR GLOBAL ANNUAL EVENT 2023
+            </h1>
+          </div>
+          <div
+            class="p-4 border-2 border-gray-100 bg-gray-50 drop-shadow-md md:w-[90%] lg:w-[60%]"
+          >
+            <p class="text-primary">
+              Motivational speeches can provide inspiration, boost morale, and
+              help individuals overcome obstacles. They offer practical tools
+              and strategies to help people achieve their goals and dreams. They
+              can also help individuals build self-confidence, improve
+              communication skills, and foster a positive mindset.
+            </p>
+          </div>
+          <div>
+            <span class="text-primary text-3xl font-[900]">
+              2023.10.15(THU) ~ 16(FRI)
+            </span>
+            <p class="mt-2">
+              @National Musem of korean art, central Nabi and more
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Program -->
+    <section id="program" class="bg-primary bg-bgProgram bg-repeat">
+      <div class="container flex flex-col space-y-8 mx-auto px-4 pb-8">
+        <div class="flex flex-col gap-4 items-center pt-8 px-2">
+          <h2 class="text-white text-2xl font-medium">Main Program</h2>
+          <span class="dash bg-callAction"></span>
+        </div>
+        <div class="flex flex-col md:flex-row gap-4">
+          <div
+            class="flex md:flex-col md:items-center gap-4 justify-between bg-gray-500 bg-opacity-20 py-4 rounded my-hover"
+          >
+            <div
+              class="flex md:flex-col w-[32%] md:w-full gap-2 justify-center items-center"
+            >
+              <img width="44" src="./images/icons4.png" alt="" />
+              <h4 class="text-callAction font-semibold">Lecture</h4>
+            </div>
+            <p class="text-secondary w-[68%] md:px-4 md:w-full text-sm">
+              Lorem ipsum dolor sit amet, consectetur adip is icing elit. Vero,
+              iste molestias? Lorem ipsum dolor sit amet.
+            </p>
+          </div>
```

Comment:
- Kindly, the div will be used only for unknown tags, so please update your program lists to `article` instead of `div` for better semantic reasons 👌

Reply:
done